style.css

```
@@ -198,41 +203,161 @@
   font-size: 12px;
 }
 
-@media (max-width: 700px) {
-  .hero-text__heading {
-    font-size: 20px;
-  }
+/* ------------courses page----- */
 
-  .nav-links__item {
-    display: block;
-  }
+.content-section {
+  padding: 4% 4%;
+}
 
-  .nav-links {
-    position: absolute;
-    background: #f44336;
-    height: 100vh;
-    width: 200px;
-    top: 0;
-    right: 0;
-    text-align: left;
-    z-index: 1;
-    transition: 0.5s;
-    display: none;
-  }
+.switcher {
+  display: inline-flex;
+}
 
-  .navbar .fa {
-    display: block;
+.courses-cards {
+  text-align: center;
+}
+
+.switcher-col {
+  margin: 8px;
+  padding: 16px;
+  font-size: 15px;
+  cursor: pointer;
+}
+
+.rounded-tabs {
+  padding: 8px;
+  border-radius: 200px;
+  background-color: #2a2c2c;
+}
+
+.rounded-tabs-link{
+padding: 8px 24px;
+    border-radius: 200px;
+    border-width: 0;
+    background-color: transparent;
     color: #fff;
-    margin: 10px;
-    font-size: 22px;
-    cursor: pointer;
-  }
+}
 
-  .nav-links__list {
-    padding: 30px;
-  }
+.current-tab{
```

```suggestion
.current-tab {
```

there are missing spaces in many places of css file